src/pages/About/Card.tsx

```diff
@@ -41,13 +44,15 @@ const Card = ({
   to: string
   external?: boolean
 }) => {
+  const isDarkMode = useIsDarkMode()
```

Comment:
this makes me feel like we should have a theme.isDarkMode so that styled components can access it directly

Reply:
hmm yea... i have been thinking this is mostly an anti-pattern. i'm doing it right now because i don't want to push back on design so late but we shouldn't need to special case like this mostly?